virtcontainers/hyperstart_agent.go

```diff
@@ -798,3 +798,8 @@ func (h *hyper) sendCmd(proxyCmd hyperstartProxyCmd) (interface{}, error) {
 
 	return h.client.HyperWithTokens(proxyCmd.cmd, tokens, proxyCmd.message)
 }
+
+func (h *hyper) onlineCPUMem() error {
+	// hyper agent onlines vCPUs and Memory automatically
```

Comment:
So our hyperstart agent assumes that CPUs are hotplugged?

Reply:
We never implemented the command `ONLINE_CPUMEM` from `cc-agent`, and I don't think we want/should spend some time on that.

Reply:
Then we should at least say that it can be implemented via hyperstart's `ONLINECPUMEM` cmd rather than leaving false information here.

Reply:
I agree.
@devimc could you please rework this comment.